server/confdata-binlog-replay.cpp

```diff
@@ -1104,6 +1113,14 @@ void confdata_binlog_update_cron() noexcept {
     return;
   }
 
+  // Force statistics update if binlog hasn't been rotated successfully in specific time
+  if (BinlogBufferWriter.unsuccessful_rotation_attempts > 0) {
+    std::chrono::seconds wait_rotation_time {time(NULL) - BinlogBufferWriter.unsuccessful_rotation_first_ts};
```

Comment:
It's a bit strange to combine old C-style `time(NULL)` with `std::chrono`, maybe it's better to use `std::chrono::system_clock::now()`